### PR TITLE
changes the mining medic jacket to be more suitable for lavaland

### PR DESCRIPTION
**Pull request by Moltijoe**
Actually using the jacket as a mining medic is sorta inting
Gives it some fire protection

If i was able to sprite, i would have just made a mining medic version of the explorer suit with different stats

:cl:  
tweak: mining medic stats are closer to that of the explorer suit
/:cl:

### CODE REVIEW

File: yogstation/code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -2,8 +2,14 @@
 /obj/item/clothing/suit/toggle/labcoat/emt/explorer
 	name = "mining medic's jacket"
 	desc = "A protective jacket for medical emergencies on off-world planets. Has MM embossed into it."
-	armor = list(melee = 10, bullet = 10, laser = 0,energy = 0, bomb = 0, bio = 50, rad = 0, fire = 50, acid = 50)
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	cold_protection = CHEST|ARMS
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
```
Comment (ShoziOffical) [inline yogstation/code/modules/clothing/suits/labcoat.dm:7]: is this a bit too much for a jacket

File: yogstation/code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
+	heat_protection = CHEST|ARMS
+	flags_prot = HIDEJUMPSUIT
+	armor = list(MELEE = 30, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 50, BIO = 100, RAD = 50, FIRE = 50, ACID = 50, WOUND = 10)
 	allowed = list(/obj/item/analyzer,/obj/item/stack/medical,/obj/item/dnainjector,/obj/item/reagent_containers/dropper,/obj/item/reagent_containers/syringe,/obj/item/reagent_containers/autoinjector,/obj/item/healthanalyzer,/obj/item/flashlight/pen,/obj/item/reagent_containers/glass/bottle,/obj/item/reagent_containers/glass/beaker,/obj/item/reagent_containers/pill,/obj/item/storage/pill_bottle,/obj/item/paper,/obj/item/melee/classic_baton/telescopic,/obj/item/soap,/obj/item/sensor_device,/obj/item/tank/internals, /obj/item/hypospray)
+	resistance_flags = FIRE_PROOF
```
Comment (ShoziOffical) [inline yogstation/code/modules/clothing/suits/labcoat.dm:12]: don't think a jacket should be fire proof

**Comment by ShoziOffical** [comment]
maybe give mining medic an explorer suit storage

**Comment by Moltijoe** [comment]
> maybe give mining medic an explorer suit storage

What id prefer to do is create a mining medic version of the explorer suit and give them that. But i cant sprite for the life of me

**Comment by ShoziOffical** [comment]
> > maybe give mining medic an explorer suit storage
> 
> What id prefer to do is create a mining medic version of the explorer suit and give them that. But i cant sprite for the life of me

you can ask someone for it

**Comment by Moltijoe** [comment]
> you can ask someone for it

That requires someone willing to do it
Maybe i'll just close this anyways, since i plan to delete the mining medic jacket if the suit ever gets made

**Comment by Moltijoe** [comment]
yeah, i'll probably just make the mining medic explorer suit rather than wasting time on this

**Pull request closed without merging**